install/troubleshooting_installation.md

```diff
@@ -96,3 +97,10 @@ for additional information on system hardware, troubleshooting, and administrati
       * Timeout on `cray-sls-init-load` during Install CSM Services due to Postgres cluster in `SyncFailed` state
 
       See [Troubleshoot Postgres Database](../operations//kubernetes/Troubleshoot_Postgres_Database.md#syncfailed)
+   <a name="csm_installation_failure"></a>
+
+   1. CSM Install Failure "admin-client-auth" Secret Not Found
+
+      If a new installation is failing with a missing admin client secret.
```

Comment:
```suggestion
      If a new installation is failing with a missing `admin-client-auth` secret, then see
      [CSM Services Install Fails Because of Missing Secret](csm_installation_failure.md).
```